### DISCUSSION TITLE
moveing robot on 3D panel with only '/joint_states'

### DISCUSSION THREAD

**Discussion by SeiyaSadashima-tch** [foxglove/community#63] [Q&A] [answered]
Hello, I am currently creating a bag file from the machine log and trying to visualize it. Do you have any ideas for moving a robot in a 3D panel with only the '/joint_states' topic?

**Comment by jtbandes** [comment, marked as answer]
Currently, only the [URDF Viewer panel](https://foxglove.dev/docs/studio/panels/urdf-viewer) supports joint states. We are planning to add this to the 3D panel as part of this project (or after initial release): https://github.com/foxglove/studio/issues/3143

**Reply by SeiyaSadashima-tch** [reply]
Thank you.
I will wait in anticipation for that function to be implemented.
